sdk/communication/azure-communication-mediacomposition/src/main/java/com/azure/communication/mediacomposition/MediaCompositionClient.java

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.mediacomposition;
+
+import com.azure.communication.mediacomposition.implementation.MediaCompositionsImpl;
+import com.azure.communication.mediacomposition.implementation.models.CompositionStreamState;
+import com.azure.communication.mediacomposition.implementation.models.MediaCompositionBody;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+
+import java.util.Objects;
+
+/**
+ * Synchronous client for Communication service media composition operations
+ */
+@ServiceClient(builder = MediaCompositionClientBuilder.class, isAsync = false)
+public final class MediaCompositionClient {
```

Comment:
Should we also update the samples in the code to use the Media Composition Client's methods then?

Reply:
That'll be separate PR. :-)